### ISSUE TITLE
1.0.0 release

### ISSUE THREAD

**Issue by jamestalmage** [olalonde/follow-redirects#23]
@olalonde 

I think it is time for a 1.0.0 release.

It gives us the ability to publish updates in a `semver` responsible way.

I have added a couple issues I want to close before then to  the 1.0.0 milestone. Once those are done, I think we should do it.

**Comment by olalonde** [comment]
Sounds good!

**Comment by akohout** [comment]
+1

**Comment by akohout** [comment]
It's hard to follow your changes (e.g. the bump from 0.0.6 to 0.0.7) at the moment. And using a 0.0.x looks like not much trust from the library authors.

**Comment by jamestalmage** [comment]
Well,
We just added a real test suite in 0.0.5 (which was just released a few months ago).

Fixing #12 and #19 is going to introduce a breaking change (though it will affect a very small minority of users being redirected in non-standard ways). That will be a good time to go to 1.0.0. 

I started https://github.com/jamestalmage/browser-redirect-test to identify what behavior we should follow to best to mimic existing browsers. I have stalled out after realizing how terribly non-conformant the current field of browsers is. Other than `303` and `307` redirect codes, there is not much agreement between browsers.
